provider/datagen/src/bin/datagen.rs

```diff
@@ -170,24 +170,11 @@ fn main() -> eyre::Result<()> {
                     "Analyzes the binary and only includes keys that are used by the binary."
                 ),
         )
-        .arg(
-            Arg::with_name("HELLO_WORLD")
-                .long("hello-world-key")
-                .help("Whether to include the 'hello world' key."),
-        )
-        .arg(
-            Arg::with_name("ALL_KEYS")
-                .long("all-keys")
-                .help("Include all keys known to ICU4X."),
-        )
         .group(
             ArgGroup::with_name("KEY_MODE")
```

Comment:
### Discussion on `--all-keys` being default

Pros and cons for making this change:

1. Pro: `--all-keys` will give you enough data that ICU4X will always work.
2. Con: Although there are some cases where you want all keys, it's certainly not the case for most average clients who are building an app with only a few components enabled. You might want all keys if you are an OS providing a dylib; a data service; or _maybe_ if you are using databake and don't care about optimizing build times. It's not clear that these cases are common enough for this to be the default option.
3. Con: Requiring callers to specify `--all-keys` makes it clearer what icu4x-datagen is doing, and also what you need to change if you no longer want all keys.

Reply:
Noted. Again I changed this for a better OOBE. Right now clients have no idea what to pass to this argument. They will need to read our tutorials (the book) to learn about keys and how to select them, but that can be done later.